buildSrc/src/main/kotlin/ort-base-conventions.gradle.kts

```diff
@@ -44,6 +44,21 @@ repositories {
             includeGroup("org.gradle")
         }
     }
+
+    exclusiveContent {
+        forRepository {
+            // com.blackducksoftware.bdio:bdio2
```

Comment:
I tried separate groups, and spent too much time without success already yesterday. In case you know how the entries should look like, please let me know. (Maybe we can do separation in a follow-up PR?)

Reply:
I believe this should work:

    exclusiveContent {
        forRepository {
            maven("https://repo.blackduck.com/bds-integrations-release")
        }

        filter {
            includeGroup("com.blackduck.integration")
            includeGroup("com.blackducksoftware.magpie")
        }
    }

    exclusiveContent {
        forRepository {
            maven("https://sig-repo.synopsys.com/bds-bdio-release")
        }

        filter {
            includeGroup("com.blackducksoftware.bdio")
        }
    }

Reply:
Cool, thanks!